### PR TITLE
Added sign-up and sign-in page and sessions

### PR DESCRIPTION
**Pull request by raptattention**
I added in a sign-in page, but I don't know how you sign out a user. How do you delete the session and the cookie when you sign-out? Also, I have trouble searching through the recipes through hapi and mongodb. How do you search through an array in a collection?

### CODE REVIEW

File: public/custom.css [hunk 1]
```diff
@@ -1,25 +1,29 @@
 .img-responsive {
```
Comment (h8rry) [inline public/custom.css:1]: You have indentation problems

**Comment by h8rry** [comment]
You have indentation problems: https://github.com/WDI-HK-8/Recipz/pull/1/files

**Comment by h8rry** [comment]
The rest looks good. Can merge after fixing indentations. :sun_with_face:

**Comment by f3r** [comment]
Please fix as per @harrychenca comments so we can merge it this morning :)

**Comment by h8rry** [comment]
@raptattention: Fixed issues in another branch. Merging this.

**Pull request merged**